flashlight/memory/managers/CachingMemoryManager.h

```diff
@@ -133,6 +139,14 @@ class CachingMemoryManager : public MemoryManagerAdapter {
 
   void tryMergeBlocks(Block* dst, Block* src, BlockSet& freeBlocks);
   void freeBlock(Block* block);
+
+ private:
+  // Non-const runtime options in order to fine tune the behavior of this
+  // manager. Prevents to recycle some buffers, to be set by the user if
+  // desired:
+  size_t recyclingSizeLimit;
```

Comment:
Our convention is to have private members' names end in underscores - e.g. `recyclingSizeLimit_`. Can you also move the initialization to inline here (e.g. `recyclingSizeLimit_{std::numeric_limits<size_t>::max()}` rather than the ctor?

Reply:
done